tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/superhero/api/HeroFinder.java

```diff
@@ -429,7 +429,7 @@ public SuperHero logLocationWithArray(@Name("name") String name,
     @Mutation
     @Description("Log the last place the hero was seen (Long Lat) using an array") 
     public SuperHero logLocationLongLatWithArray(@Name("name") String name,
-                                     @JsonbNumberFormat("00.0000000 'longlat'") 
+                                     @JsonbNumberFormat(value = "00.0000000 longlat", locale = "en-GB") 
```

Comment:
Does removing the single-quotes around "longlat" cause issues with the schema test?

Reply:
Nope. Only need it escape special characters, so I removed it. Both 8 and 11 still pass